golang.go

```diff
@@ -0,0 +1,19 @@
+// pacage docummentation
```

Comment:
Typo inside comment in word `docummentation`. Possible corrections:  `documentation` (93)
Typo inside comment in word `pacage`. Possible corrections:  `package` (29), `packages` (0), `manage` (0)